system/Commands/Server/rewrite.php

```diff
@@ -24,7 +24,9 @@
     return;
 }
 
-$uri = urldecode(parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH));
+$uri = urldecode(
+    parse_url('http://ci4.dev' . $_SERVER['REQUEST_URI'], PHP_URL_PATH) ?? ''
```

Comment:
Why not take the free advertising?
```suggestion
    parse_url('https://codeigniter.com' . $_SERVER['REQUEST_URI'], PHP_URL_PATH) ?? ''
```

Reply:
Done. 👍